Client/FeatureFlags/FeatureFlagsManager.swift

```diff
@@ -107,16 +109,18 @@ class FeatureFlagsManager: HasNimbusFeatureFlags {
         let feature = NimbusFlaggableFeature(withID: convertCustomIDToStandard(featureID),
                                              and: profile)
         switch featureID {
-        case .startAtHome:
-            if let option = desiredState as? StartAtHomeSetting {
-                feature.setUserPreference(to: option.rawValue)
-            }
+        case .onboardingNotificationCard: return
```

Comment:
Yes, it's not settable, however all feature flags with custom ids must be represented or otherwise we'll get a build error. That's why `wallpaperVersion` is in the switch as well. I am not sure throwing is a good option here - what benefit would it give us over ignoring the set call?